test/Distributed/Runtime/distributed_actor_self_calls.swift

```diff
@@ -0,0 +1,75 @@
+// RUN: %target-swift-emit-silgen -enable-experimental-distributed -disable-availability-checking -parse-as-library %s | %FileCheck %s
+// RUN: %target-run-simple-swift(-Xfrontend -enable-experimental-distributed -Xfrontend -disable-availability-checking -parse-as-library)
+
+// REQUIRES: executable_test
+// REQUIRES: concurrency
+// REQUIRES: distributed
+
+// rdar://76038845
```

Comment:
Since it's opaque to most people, I suggest removing this radar reference from this file. Instead, limit these references to commit messages / PRs, so there are still breadcrumbs for those with access.

Reply:
Hmm, this specific pattern of those 3 comments we have "everywhere" in 60 files already so want to stay consistent with that...

I do agree though that it's annoyingly opaque, I never know if we can just paste the title at least maybe too 🤔

Reply:
Ideally, describing what the purpose of the test is and what it covers would be better. Just because other files contain a radar number doesn't mean that it's a good practice. If a contributor breaks this regression test, how can we help them know what is being tested?

Reply:
Yeah, I don't disagree -- I guess we should clean that up in all of concurrency